src/lib/viewers/media/MediaControls.js

```diff
@@ -611,7 +611,9 @@ class MediaControls extends EventEmitter {
 
         if (this.wrapperEl && this.wrapperEl.parentNode) {
             this.wrapperEl.parentNode.classList.remove(SHOW_CONTROLS_CLASS);
-            this.filmstripHideHandler();
+            if (this.filmstripEl) {
```

Comment:
Any thoughts on putting this check inside of the handler? That way filmstripHideHandler can be a safe function itself.